radixdlt/src/integration/java/com/radixdlt/consensus/CrashFaultNetworkTest.java

```diff
@@ -17,52 +17,203 @@
 
 package com.radixdlt.consensus;
 
-import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 import com.radixdlt.crypto.ECKeyPair;
+import com.radixdlt.crypto.ECPublicKey;
+import com.radixdlt.examples.tictactoe.Pair;
 import io.reactivex.rxjava3.core.Observable;
+import org.assertj.core.api.Condition;
+import org.junit.Test;
+
 import java.util.Arrays;
+import java.util.HashSet;
 import java.util.List;
+import java.util.Random;
+import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
-import org.junit.Test;
+
+import static org.assertj.core.api.AssertionsForInterfaceTypes.assertThat;
 
 /**
- * Tests with networks with crashed nodes
+ * Tests with networks with crash-stopped nodes (that do not recover).
+ * These tests comprise both static and dynamic configurations (where nodes crash before or during runtime, respectively).
  */
 public class CrashFaultNetworkTest {
-	static List<ECKeyPair> createNodes(int numNodes) {
+	static List<ECKeyPair> createNodes(int numNodes, String designation) {
 		return Stream.generate(() -> {
-			return ECKeyPair.generateNew();
+			// efficient? no. useful? yes.
+			ECKeyPair keyPair;
+			do {
+				keyPair = ECKeyPair.generateNew();
+			} while (!keyPair.euid().toString().startsWith(designation));
```

Comment:
POW?!

Reply:
Essentially, yes, though with a *super* low "difficulty" and for a good reason (interpretability of results and logs). I would like a more proper feature for this but we don't have one yet, so it's a placeholder.

Reply:
After some more changes this isn't that useful anymore, so I've removed it for now